### ISSUE TITLE
Magic link flow emphasis

### ISSUE THREAD

**Issue by jaclync** [woocommerce/woocommerce-ios#7437]
Project thread: pe5sF9-iy-p2
Design: hHmVq03kKpvTDzV3uV4wqw-fi-14%3A2092

I'm still waiting for clarifications on the 3 iterations, following the Android PRs for now:
- https://github.com/woocommerce/woocommerce-android/pull/7079
- https://github.com/woocommerce/woocommerce-android/pull/7124
- https://github.com/woocommerce/woocommerce-android/pull/7127

## Iteration 1

https://github.com/woocommerce/woocommerce-ios/pull/7449

<img width="664" alt="Screen Shot 2022-08-10 at 5 16 52 PM" src="https://user-images.githubusercontent.com/1945542/183864300-bfbd3ffe-8d9f-4a96-ac8d-cf6ff829bee7.png">

## Iteration 2

<img width="702" alt="Screen Shot 2022-08-12 at 1 46 09 PM" src="https://user-images.githubusercontent.com/1945542/184292311-51896409-baa2-47bb-b369-2eb3adb293fa.png">

## Iteration 3

In conflict with Iteration 1.

**Comment by pmusolino** [comment]
@jaclync could we close this issue? Or there is still something missing here?

**Comment by jaclync** [comment]
> could we close this issue? Or there is still something missing here?

Hey @pmusolino there is Iteration 3 pe5sF9-iy-p2#iteration-3-1, but it conflicts with Iteration 1 and we only got to implmenet Iteration 1 due to the lack of A/B test setup as in Android. I just checked the Android A/B experiment results, and it looks like Iteration 3 (Variant B) didn't even perform as well as the baseline (no changes). Does it make sense to skip Iteration 3 in this case?
